internal/external/k8s/client.go

```diff
@@ -9,21 +9,35 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . Client
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
 
 // Client is a kubernetes client interface used internally. It copies functions from
 // sigs.k8s.io/controller-runtime/pkg/client
+//
+//counterfeiter:generate . Client
 type Client interface {
 	Create(ctx context.Context, obj client.Object, opts ...client.CreateOption) error
 	Get(ctx context.Context, key client.ObjectKey, obj client.Object) error
+
 	Update(ctx context.Context, obj client.Object, opts ...client.UpdateOption) error
 	Delete(ctx context.Context, obj client.Object, opts ...client.DeleteOption) error
 	DeleteAllOf(ctx context.Context, obj client.Object, opts ...client.DeleteAllOfOption) error
 	List(ctx context.Context, obj client.ObjectList, opts ...client.ListOption) error
 	Patch(ctx context.Context, obj client.Object, patch client.Patch, opts ...client.PatchOption) error
 
-	Status() client.StatusWriter
-
 	RESTMapper() meta.RESTMapper
 	Scheme() *runtime.Scheme
+
+	Status() client.StatusWriter
+}
+
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . StatusWriter
```

Comment:
You don't need this line since you're using the `counterfeiter:generate` directive with the line 12 change.